scrapers/EliteBabes.yml

```diff
@@ -0,0 +1,50 @@
+name: EliteBabes
+performerByName:
+  action: scrapeXPath
+  queryURL: https://www.elitebabes.com/?s={}
+  scraper: performerSearch
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - www.elitebabes.com/model/
+    scraper: performerScraper
+
+xPathScrapers:
+  performerSearch:
+    performer:
+      Name: //article[@id='content']/ul[@class='gallery-a b d']/li/a[contains(@href,"model/")]/span/text()
+      URL:
+        selector: //article[@id='content']/ul[@class='gallery-a b d']/li/a[contains(@href,"model/")]/@href
+
+  performerScraper:
+    common:
+      $infoPiece: //div/ul[@class='list-aside hidden']/li/span
+    performer:
+      Name: //div/h2[@class='fn']/text()
+      Birthdate:
+        selector: $infoPiece[text() = 'Born:']/../text()
```

Comment:
Going to need some postProcess here to handle the birth date. Unfortunately they are not consistent

https://www.elitebabes.com/model/mango-a/ has `1989-12-11`
https://www.elitebabes.com/model/mila-azul/ has `1997`
https://www.elitebabes.com/model/alisa-i/ has `March 1997`

@bnkai I may be behind in the times, do we have a method of handling multiple possibilities of birth dates? Or is that only something a script could handle?

Reply:
In a couple of scrapers i think we use a series of `parseDate`. If a format doesnt match parseDate will print an error and return the original value and then continue to the next so no problem with using multiples.
The main issue from the referenced urls @halorrr is the lack of full date. Do we want to regex replace missing months/days with 01 or not? (The same issue is present in some movie scrapers). It is different having a birthdate in 1997, the birthdate actually being 1997-01-01 and the birthdate being regex replaced to 1997-01-01. 
Leaving just what is available means that the user has to decide and make a manual edit (maybe adding a comment/tag somewhere  that the data is not accurate) so that he can save.
Replacing, filling the date means that the user doesnt need to do anything but also doesnt know that the date was prefilled and thus not accurate.

Reply:
Good point, I'm thinking it is safe to parse the full date format, and let anything else fall through to the user to adjust